README.md

```diff
@@ -1,69 +1,22 @@
-# City Scrapers Template
+# City Scrapers Philly
 
 [![CI build status](https://github.com/City-Bureau/city-scrapers-philly/workflows/CI/badge.svg)](https://github.com/City-Bureau/city-scrapers-template/actions?query=workflow%3ACI)
 [![Cron build status](https://github.com/City-Bureau/city-scrapers-philly/workflows/Cron/badge.svg)](https://github.com/City-Bureau/city-scrapers-template/actions?query=workflow%3ACron)
 
-Template repo for creating a [City Scrapers](https://cityscrapers.org/) project in your area to scrape, standardize and share public meetings from local government websites. You can find more information on the [project homepage](https://cityscrapers.org/) or in the original City Scrapers repo for the Chicago area: [City-Bureau/city-scrapers](https://github.com/City-Bureau/city-scrapers).
+Repo for the [City Scrapers](https://cityscrapers.org) project in Spokane, Washington.
```

Comment:
Isn't this repo for Philadelphia?

Reply:
Hah, it is indeed! Dumb error. Thanks for catching. I copied and pasted text from `city-scrapers-philly` and must have overlooked that part.